uit/gui_tools/connect.py

```diff
@@ -77,7 +77,13 @@ def __init__(self, **params):
             name='HPC System',
         )
 
-    @param.depends('system', 'uit_client', watch=True)
+    @param.depends('uit_client', watch=True)
+    def update_system_options(self):
+        if self.uit_client is not None:
+            self.param.system.objects = self.uit_client.systems
+            self.system = self.uit_client.systems[0]
```

Comment:
This will sort the dropdown in "Connect to HPC System":
```suggestion
            self.param.system.objects = sorted(self.uit_client.systems)
            self.system = sorted(self.uit_client.systems)[0]
```

Reply:
@araglu or should `systems` be sorted at the `Client` level?

Reply:
see https://github.com/erdc/pyuit/pull/50/commits/40ca77d667a6b9bc395de3ac1cc4b9853d240cb3

Reply:
That works even better! I can't think of any reason we would need to keep UIT+'s original order.